docs/examples/data_mover.md

```diff
@@ -0,0 +1,156 @@
+<h1 align="center">Stateful Application Backup/Restore - VolumeSnapshotMover</h1>
+<h2 align="center">Relocate Snapshots into your Object Storage Location</h2>
+
+<h2>Prerequisites:<a id="pre-reqs"></a></h2>
+
+<hr style="height:1px;border:none;color:#333;">
+
+- Have a stateful application running in a separate namespace. 
+
+- Have an appropriate StorageClass and VolumeShapshotClass. **Make sure there is only one default of each.**
+  - Include the label `velero.io/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` 
+  to let Velero know which to use.
+  - `deletionPolicy` must be set to `Retain`
+
```

Comment:
If possible I would like to redirect to CSI section for requirements.. so that effort regarding snapshotclass label are not duplicated.